pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/TopicsImpl.java

```diff
@@ -1239,6 +1251,13 @@ private WebTarget namespacePath(String domain, NamespaceName namespace, String..
         return namespacePath;
     }
 
+    private WebTarget topicPath(TopicName topic, Map<String, String> metadata, String... parts) {
+        final WebTarget base = metadata != null ? adminV3Topics : (topic.isV2() ? adminV2Topics : adminTopics);
```

Comment:
so this is where we select the "version" of the API for the topics.

can you please add a comment here in order to give more context ?
in the future we will have to use the v3 API for new features.

Reply:
Yes, when user create topics with properties, we will invoke v3.

Reply:
done, thanks